.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+
+jobs:
+
+  tests:
+    uses: ./.github/workflows/tests.yml
+
+  publish:
+    if: github.ref == 'refs/heads/master'
+    needs: [tests]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: 'master'
+      - uses: ./.github/actions/dependencies
+        with:
+          root: false
+          packages-react: true
+          packages-styles: true
+      - name: Build packages
+        run: |
+          NODE_ENV=production yarn --cwd packages/react build
+          NODE_ENV=production yarn --cwd packages/styles build
+      - name: Publish @deque/cauldron-styles
+        run: |
+          cd packages/styles
+          PACKAGE_VERSION="$(npm pkg get version | tr -d \")"
+          npm version "$PACKAGE_VERSION-canary.${GITHUB_SHA:0:8}" --allow-same-version --no-git-tag-version
+          npm publish --tag=next
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+      - name: Publish @deque/cauldron-react
+        run: |
+          cd packages/react
+          PACKAGE_VERSION="$(npm pkg get version | tr -d \")"
+          npm version "$PACKAGE_VERSION-canary.${GITHUB_SHA:0:8}" --allow-same-version --no-git-tag-version
```

Comment:
Should this be in the `canary` job instead of the `publish` job?

Reply:
Whoops, good catch! 🤦 

Looks like I swapped them around accidentally.